pyproject.toml

```diff
@@ -8,6 +8,6 @@ dependencies = [
 [project.optional-dependencies]
 dev = [
     "packaging~=23.1",
-    "pip-tools>=7.3.0,<8.0.0",
+    "uv>=0.1.24",
```

Comment:
It still feels weird to me to include `uv` in the dependencies since you'd be installing the dependencies with `uv` itself and this doesn't pin a version anyway.

Would you mind omitting it for now and we can add it later if there are problems?

Reply:
That makes sense. Just pushed that change
(also can't believe I spelled omit as emit)